help/develop/domains/chem/cheminformatics.md

```diff
@@ -109,38 +86,34 @@ columns ([code sample](https://public.datagrok.ai/js/samples/domains/chem/simila
 Produces a Datagrok [`DataFrame`](https://datagrok.ai/js-api/classes/dg.DataFrame) with a
 single [`Column`](https://datagrok.ai/js-api/classes/dg.Column), where the i-th element contains a similarity score for
 the i-th element of the
-input `Column` ([code sample](https://public.datagrok.ai/js/samples/domains/chem/similarity-scoring-scores))
-.
+input `Column` ([code sample](https://public.datagrok.ai/js/samples/domains/chem/similarity-scoring-scores)).
 
-Similarly to the substructure-library based method of `searchSubstructure`, these functions maintain a cache of
+Similarly to the `searchSubstructure` function, these functions maintain a cache of
 pre-computed fingerprints. Once the function `findSimilar` or `getSimilarities` meets the `column` previously unmet, it
-builds the cache of computed fingerprints for the molecules of this `column`, and later uses it to compute similarity
+builds the cache of computed fingerprints for the molecules in `column`. Later it is used to compute similarity
```

Comment:
Same comment for later. See suggestions above.

Reply:
Done